src/encoded/static/components/filegallery.js

```diff
@@ -533,6 +572,9 @@ class RawSequencingTable extends React.Component {
 
                                     return (
                                         <tr key={file['@id']}>
+                                            {i === 0 ?
+                                              <td rowSpan={groupFiles.length} className={`${bottomClass} merge-right table-raw-merged table-raw-biorep`}>{groupFiles[0].biological_replicates[0]}</td>
+                                              : null}
```

Comment:
We don’t (and AirBnB doesn’t) have a standard for how to indent conditional expressions, but we do need to enforce four-space indents. I’m surprised eslint doesn’t mark this.

Reply:
Maybe an .editorconfig file will help with catching editor-specific things eslint misses. I will dig into that when I get bandwidth. I pushed the 4 indent. Thanks for keeping an eye on that.